boilerplate/package.json.ejs

```diff
@@ -31,8 +31,8 @@
     "@react-native-community/async-storage": "^1.5.1",
     "apisauce": "1.0.3",
     "lodash.throttle": "4.1.1",
-    "mobx": "^4.13.0",
-    "mobx-react": "^5.4.4",
+    "mobx": "^5.13.0",
```

Comment:
```suggestion
    "mobx": "^4.13.0",
```

We need to lock `mobx` at 4.x because of https://github.com/facebook/hermes/issues/28 (as well as the old version of JavaScriptCore in iOS 9 not supporting Proxies). We can move to 5.x if we are okay with not supporting iOS 9 and Hermes, but I don't think that's a viable path forward at this time.

Reply:
Got it - I'll keep it at 4.13.0.